packages/es-components/src/components/containers/popover/Popup.js

```diff
@@ -197,10 +197,9 @@ function Popup(props) {
               placement={position}
               modifiers={{
                 preventOverflow: { boundariesElement: document.body },
-                hide: { enabled: true },
                 flip: {
                   enabled: !disableFlipping,
-                  behavior: ['left', 'right', 'top', 'bottom']
+                  behavior: ['left', 'right', 'top', 'bottom', 'top']
```

Comment:
Is 'top' supposed to be in here twice?

Reply:
Yes, because of how popper handles flipping, a bottom oriented popover wont flip to the top without the second 'top'.